tests/UI/expected-screenshots/ContainerTag_edit_updated_back_to_list.png.orig

```diff
@@ -0,0 +1,8 @@
+version https://git-lfs.github.com/spec/v1
```

Comment:
@AltamashShaikh this file will need to be deleted there is a merge conflict

Reply:
deleted